src/index.spec.ts

```diff
@@ -451,6 +452,12 @@ describe('reflection', () => {
     expectLiteralField(InstanceOf(Test), 'tag', 'instanceof');
     expectLiteralField(Dictionary(Array(InstanceOf(Test))), 'tag', 'dictionary');
   });
+
+  it('brand', () => {
+    const C = Number.withBrand('someNumber') as Brand<'someNumber', Number>;
```

Comment:
Why cast here?

Reply:
That was just nothing... fixed. Thanks 🌟